### ISSUE TITLE
Problem with Pimatic via WLAN

### ISSUE THREAD

**Issue by Pedder007** [pimatic/pimatic#1109]
Hi all,

I just set up my new Pimatic-Homeduino over the last weeks, which meanwhile works well.
These days I plugged off the ethernet cable and would like to continue to work via the integrated WLAN functionalitiy of my Raspberry3B, but ....:
The web-front is shown in my browser (the RB is also reachable via RDP), but I'm not able to trigger any activities via the web-frontend! Also temperature data is not shown any longer. The frontend seems to work normal, switching buttons, changing groups, etc., but no command is really executed.
When I plug in the ethernet cable in parallel, everything works fine, also via the WLAN connection. When I plug off again, same effect as described...?
I also switched the static IP-address to the wlan-connect, but this didn't help.
Has anybody an idea what this can be?

Greating Pedder007 (=Pedder on Pimatic Forum)

**Comment by Pedder007** [comment]
An Update:
I tested again. Deleted the ethernet-entry from my router and asigned again the former static IP from the eth-entry to the WLAN entry.
Upfront I stopped Pimatic and then completely rebooted the raspberry, while plugging off the eth cabel.
After reboot:
- ifconfig shows up with the correct IP for wlan0 and eth0 has no IP
- RDP session from Win-Client runs
- Pimatic runs and offers the WEB-GUI, now incl. showing the correct temperature dates
- But still all buttons in the GUI seems only to be dummies :-(
- I tested also to switch by rule, with the result, that again only the 'dummy-button' changed his status but no order seems to be send out any longer to the receivers.
- I really ask myself, what the WLAN-contect, while the GUI itself works and also temperature data is shown, has to to with the functionality of the switches/shutters/etc. ..?

No one here with an idea?
